mysql-test/suite/innodb/t/mysqld_core_dump_without_large_mem_buf.test

```diff
@@ -0,0 +1,22 @@
+################################################################################
+# This test is to test if mysqld can dump a core without large memory buffers.
+# See opt file for the config:
+#   (1) --innodb-dump-core-without-large-mem-buf is set
+#   (2) the buffer pool is set to be large so that without dropping the large
+#       memory buffers the core size will be much greater than 3.5GB (the actual
+#       core size is less than 3GB now but set the limit to 3.5GB here in case
+#       the memory footprint increases in the future)
+
+--source include/not_valgrind.inc
+--source include/linux_core_pattern.inc
+
```

Comment:
Consider making this test `--source include/big_test.inc` as it request 60GB buffer pool